lib/Units.pm

```diff
@@ -43,7 +43,7 @@ our %fundamental_units = (
 	'rad'    => 0,
 	'degC'   => 0,
 	'degF'   => 0,
-	'degK'   => 0,
+	'K'      => 0,
```

Comment:
Do you want to make the corresponding change in the comments above this? (Or maybe just remove that comment block?)

Reply:
Thanks for pointing that out.  I didn't see that.  I decided to delete that comment block.  I don't see a need to have the code also in a comment block.